syntax = "proto3";

package media;

// Define the service for media operations
service MediaService {
    // Method to download media by ID
    rpc DownloadMedia (MediaRequest) returns (MediaResponse);
}

// Message to request media download
message MediaRequest {
    string media_id = 1;
    string media_type = 2; // e.g., "image", "video", "audio"
}

// Message to respond with media data
message MediaResponse {
    bytes media_data = 1;
    string media_type = 2;
}
